apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.cs

```diff
@@ -233,8 +233,7 @@ public override int GetHashCode()
         {
             unchecked
             {
-                int hashCode = StructMembers?.GetHashCode() ?? 0;
-                hashCode = (hashCode * 397) ^ (int) TypeCode;
+                int hashCode = (int) TypeCode;
```

Comment:
What was the bug here? Why is StructMembers not included anymore if it is still compared in `Equals`?